src/Controllers/gdprdelete.php

```diff
@@ -0,0 +1,38 @@
+#!/usr/local/bin/php -q
+<?php
+
+use \MyRadio\Config;
+use \MyRadio\Database;
+use \MyRadio\MyRadioException;
+use \MyRadio\MyRadio\AuthUtils;
+use \MyRadio\MyRadio\URLUtils;
+use \MyRadio\MyRadio\MyRadioForm;
+use \MyRadio\MyRadio\MyRadioFormField;
+use \MyRadio\MyRadio\MyRadioDefaultAuthenticator;
+use \MyRadio\ServiceAPI\MyRadio_User;
+
+ini_set('log_errors', 1);
+ini_set('error_log', '/tmp/php-mailparser-error.log');
+ini_set('display_errors', 'On');
+
+require_once __DIR__.'/root_cli.php';
```

Comment:
no other controllers have these, why is it here?

Reply:
Requests are put through something using the databse i don't know. its safe tho.

Reply:
Yeah, there's a folder for scripts, or should these be daemons anyway??

Reply:
is this not the scripts folder? I did check the script can't be run through get request.